Android/app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,74 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <EditText
+        android:id="@+id/et_login_username"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="10"
+        android:hint="@string/et_hint_username"
+        android:inputType="textPersonName"
+        android:minHeight="48dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.497"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.411"
+        android:autofillHints="" />
+
+    <EditText
+        android:id="@+id/et_login_password"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="10"
+        android:hint="@string/et_hint_password"
+        android:inputType="textPassword"
+        android:minHeight="48dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.497"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/et_login_username"
+        app:layout_constraintVertical_bias="0.077"
+        android:autofillHints="" />
+
+    <Button
+        android:id="@+id/btn_login"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/btn_login"
+        android:backgroundTint="@color/blue_yonder"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/et_login_password"
+        app:layout_constraintVertical_bias="0.107" />
+
     <TextView
+        android:id="@+id/textView"
+        android:layout_width="93dp"
```

Comment:
We can make the `dimens.xml` to store values like width and such, the file should be in the same folder as `colors.xml`. Our `dimens.xml` can look something like this:
```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <dimen name="btnMargin">20dp</dimen>
    <dimen name="btnPadding">10dp</dimen>
    <dimen name="btnTextSize">20sp</dimen>
    <dimen name="infoTextSize">30sp</dimen>
    <dimen name="listMargin">0dp</dimen>
    <dimen name="listItemMargin">15dp</dimen>
    <dimen name="listItemMarginLeft">10dp</dimen>
    <dimen name="textFieldHeight">100dp</dimen>
    <dimen name="textFieldMargin">5dp</dimen>
    <dimen name="textFieldWidth">300dp</dimen>
    <dimen name="textSizeBig">20sp</dimen>
    <dimen name="textSizeMiddle">15sp</dimen>
    <dimen name="welcomeTextFieldMarginBottom">120dp</dimen>
    <dimen name="dialogPadding">40dp</dimen>
</resources>
```